### PR TITLE
Enabled multi-selection of Controlled legends

### PR DESCRIPTION
**Pull request by Anush2303**
## New Behavior

Enabled Multi selection behaviour of Controlled legends for LineChart.
**Upcoming changes**: enable it for other charts.

## Related Issue(s)

- Fixes #

### CODE REVIEW

File: change/@fluentui-react-charting-4c197bf1-3dca-480d-aa17-d97a170246ff.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-charting-4c197bf1-3dca-480d-aa17-d97a170246ff.json:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33479&commitId=62884cd4bf7745063353603a3d8d1a04de952ea1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Right top edge.chromium.png|1127| Changed|
|Callout.Left top edge.chromium.png|1949| Changed|
|Callout.Root.chromium.png|2309| Changed|
|Callout.Left center.chromium.png|2616| Changed|
|Callout.No beak.chromium.png|2306| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|
|Callout.Gap space 25.chromium.png|2309| Changed|

</details>


  <details><summary><b>Coachmark</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Coachmark.Collapsed.chromium.png|159| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/charts/react-charting/src/components/LineChart/LineChart.base.tsx [hunk 2]
```diff
@@ -379,10 +379,21 @@ export class LineChartBase extends React.Component<ILineChartProps, ILineChartSt
     return domainNRangeValue;
   };
 
-  private _injectIndexPropertyInLineChartData = (lineChartData?: ILineChartPoints[]): LineChartDataWithIndex[] | [] => {
+  private _injectIndexPropertyInLineChartData = (
+    lineChartData?: ILineChartPoints[],
+    isFilterSelectedLegends?: boolean,
```
Comment (AtishayMsft) [inline packages/charts/react-charting/src/components/LineChart/LineChart.base.tsx:384]: make this as default = false. you wont need to change the existing code.

Comment (Anush2303) [reply packages/charts/react-charting/src/components/LineChart/LineChart.base.tsx:384]: oh yes.

File: change/@fluentui-react-charting-4c197bf1-3dca-480d-aa17-d97a170246ff.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  "type": "patch",
+  "comment": "Enabled Multi Select behaviour of Controlled legends",
```
Comment (AtishayMsft) [inline change/@fluentui-react-charting-4c197bf1-3dca-480d-aa17-d97a170246ff.json:3]: >Controlled [](http://example.com/codeflow?start=48&length=10)

Make corresponding change in declarative chart and fluent_line_chart.json schema example.

Comment (Anush2303) [reply change/@fluentui-react-charting-4c197bf1-3dca-480d-aa17-d97a170246ff.json:3]: done and tested.

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**